docs/Config.md

```diff
@@ -619,7 +619,7 @@ keybinding:
     viewResetOptions: D
     fetch: f
     toggleTreeView: '`'
-    openMergeTool: M
+    openMergeOptions: M
```

Comment:
This is a breaking change (users might have changed the keybinding in their config file). We have the technology to migrate users's config files to account for this, and we should do that by adding an entry to [this array](https://github.com/jesseduffield/lazygit/blob/cb2b5c37387b5dc44a802ad92ab7473d8d7e5374/pkg/config/app_config.go#L268).